pkg/storage/google_cloud_storage.go

```diff
@@ -50,20 +50,14 @@ type GoogleCloudStorage struct {
 }
 
 // NewGoogleCloudStorage creates a new GoogleCloudStorage client.
-func NewGoogleCloudStorage(ctx context.Context, opts ...Option) (*GoogleCloudStorage, error) {
+func NewGoogleCloudStorage(ctx context.Context, _ Options) (*GoogleCloudStorage, error) {
```

Comment:
The `Options` type seems to never be used; can it be removed? I may have just missed it though...

Reply:
Ya, put the cart before the horse. Removed.